recipes/ecmwf-api-client/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: ecmwf-api-client
+  version: "1.4"
+
+source:
+  fn: ecmwf-api-client-python.tgz
+  url: https://software.ecmwf.int/wiki/download/attachments/56664858/ecmwf-api-client-python.tgz
+  sha256: 423331f2ace3b247ff0d7db8c96952003136b5859ad0840a045c311ee857bcfb
+
+build:
+  number: 0
```

Comment:
@Ohjeah you can add a line here with the install instructions.

```yaml
script: python setup.py install --single-version-externally-managed --record record.txt
```

Note that when the software uses setuptools we add the extra args `--single-version-externally-managed --record record.txt` to ensure the installed files play nice with external packaging tools.